tensorflow_encrypted/protocol/pond.py

```diff
@@ -905,16 +918,8 @@ def expand_dims(self):
     def reshape(self, shape: List[int]) -> 'PondTensor':
         return self.prot.reshape(self, shape)
 
-    def broadcast(self, to: List[int]) -> 'PondTensor':
```

Comment:
any reason to delete these? just b/c they aren't implemented via dispatch?

Reply:
same question re where & gather_nd

Reply:
only because the code seemed a bit too specific, making assumptions about the internals of the backing tensors (would fail for int100 for instance)